eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -176,4 +176,24 @@ function GetValueSafelyFrom-Yaml {
   }
 
   return [object]$current
-}
\ No newline at end of file
+}
+
+function Split-ArrayIntoBatches {
```

Comment:
FYI: We just did a very similar implementation in the specs repo at https://github.com/Azure/azure-rest-api-specs/blob/main/eng/scripts/Array-Functions.ps1. This will definitely be a chance to share this logic as I'm also adding eng/common to the specs repo.

Reply:
Yeah. I think this one is a _little_ bit more generic than that one (I haven't examined the shard handling).